SimpleKeychain/A0SimpleKeychain.m

```diff
@@ -356,12 +356,12 @@ - (NSDictionary *)queryFetchOneByKey:(NSString *)key message:(NSString *)message
                                       (__bridge id)kSecAttrAccount: key,
                                       }];
 #if TARGET_OS_IPHONE
-    if (self.useAccessControl && floor(NSFoundationVersionNumber) > NSFoundationVersionNumber_iOS_7_1) {
-        if (message) {
+    if (self.useAccessControl) {
+        if (message && floor(NSFoundationVersionNumber) > NSFoundationVersionNumber_iOS_7_1) {
             query[(__bridge id)kSecUseOperationPrompt] = message;
+        } else {
+            NSLog(@"Error trying to access to non available kSecUseOperationPrompt in iOS7");
```

Comment:
@grasmeyer the change in the PR won't print anything unless accessControl is `YES`.
@tupakapoor I am merging this as it is, no point wasting time for something for such an older iOS version. Thanks for the PR.